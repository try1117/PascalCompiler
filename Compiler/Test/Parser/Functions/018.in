program test;

function a(i: integer): array[1..3] of record name: string; end;
var tmp: array[1..3] of record nAmE: string; end;
begin
  exit(tmp);
end;

begin
end.
