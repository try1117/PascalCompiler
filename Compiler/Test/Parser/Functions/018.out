test : function()
   resultType : Nil

test declarations:
   a : function(
      i : Integer
   ) resultType : Array [1, 3] of Record
      name : String
   end

   a declarations:
      tmp : Array [1, 3] of Record
         nAmE : String
      end

   |-- Statements
   |            |-- exit
   |            |      |-- tmp

